src/main/java/edu/hm/hafner/analysis/parser/PolyspaceParser.java

```diff
@@ -44,18 +44,23 @@ private Report parse(final Stream<String> lines) {
             Report report = new Report();
             Iterator<String> lineIterator = lines.iterator();
 
+            String header = "";
+            if (lineIterator.hasNext()) {
+                header = lineIterator.next();
+            }
+
```

Comment:
move into a method `readHeader`

Reply:
Done.

Reply:
In Java you normally do not use variables in such a case, see next method.